nautobot/circuits/api/nested_serializers.py

```diff
@@ -8,8 +8,21 @@
     "NestedCircuitTerminationSerializer",
     "NestedCircuitTypeSerializer",
     "NestedProviderSerializer",
+    "NestedProviderNetworkSerializer",
 ]
 
+#
+# Provider Networks
+#
+
+
+class NestedProviderNetworkSerializer(WritableNestedSerializer):
+    url = serializers.HyperlinkedIdentityField(view_name="circuits-api:providernetwork-detail")
+
+    class Meta:
+        model = Provider
```

Comment:
souldn't this be `ProviderNetwork`?
```suggestion
        model = ProviderNetwork
```